Project.toml

```diff
@@ -10,6 +10,8 @@ LoopVectorization = "bdcacae8-1622-11e9-2a5c-532679323890"
 OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 TiledIteration = "06e1c1a7-607b-532d-9fad-de7d9aa2abac"
+MappedArrays = "dbb5928d-eab1-5f90-85c2-b9b0edb7c900"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
```

Comment:
We found it troublesome to maintain the compatibility across all packages, thus we deliberately kept these two packages out to let `ImageCore` maintain the compatibility for these utilities.

Thus instead of `using MappedArrays`, you can do `using ImageCore.MappedArrays`. Same to `FixedPointNumbers`, `Colors`, `ColorTypes`, etc.